app/assets/stylesheets/modules/article.scss

```diff
@@ -23,6 +23,16 @@
       display: none;
     }
   }
+
+  .section-heading h3 {
+    border-bottom: 1px solid $sul-h2-border-color;
+    color: $sul-h2-font-color;
+    font-weight: 300;
+    font-size: 1.6em;
+    letter-spacing: 0;
+    line-height: 1.2em;
+    margin: .5em 0;
+  }
```

Comment:
This lovely styling is so that these article `<h3>`s can look the same as catalog `<h2>`s. 😬

Reply:
Is this an opportunity for some scss trickery (assuming this'd benefit from DRYing out?)

Reply:
I'm a little hesitant to bake some SCSS trickery into this when there's some additional CSS header changes in #2024

Reply:
I commented in the wrong place, but this change is not required.